### PR TITLE
New: Update new service endpoints

### PR DESCRIPTION
**Pull request by utsavized**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

*DO NOT MERGE THIS* until we are ready to swap current services with Azure functions in production/staging.

Changes:
- Update service endpoints to point to Azure functions
- Change post to use form encoded instead of multipart

### CODE REVIEW

**Comment by sarvaje** [comment]
@molant this is ready to review/squash.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -20,14 +21,20 @@ jobs:
     displayName: 'Test'
   - script: ./helpers/update-site.sh
     displayName: 'Deploy'
-    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    condition: and(succeeded(), in(variables['Build.SourceBranch'], 'refs/heads/master', 'refs/heads/functions/serverless'))
```
Comment (molant) [inline azure-pipelines.yml]: Is this to test that everything is working? Should we remove it now (or eventually)?

**Comment by molant** [comment]
@sarvaje feel free to merge whenever. I have a sick baby today so I'm not sure how much work I'll be able to get done.

File: azure-pipelines.yml (hunk 1 above)
Comment (sarvaje) [reply azure-pipelines.yml]: I'm going to restore this before merge.

**Pull request merged**